test/jdk/javax/swing/ToolTipManager/bug5078214.java

```diff
@@ -59,11 +59,11 @@ public class bug5078214 {
 
     public static void main(String[] args) throws Exception {
         try {
-            bounds = getGraphicsConfig().getBounds();
-            if (bounds == null) {
+            if (getGraphicsConfig() == null) {
                 throw new SkippedException("We need at least one screen " +
                         "with the taskbar at the bottom position.");
             }
+            bounds = getGraphicsConfig().getBounds();
```

Comment:
It's better to use the pattern that was used before: assign the result of `getGraphicsConfig()` to `testConfig` because `getGraphicsConfig()` is not a simple getter which returns the value of a field. On the other hand, it's more like premature optimisation; it's unlikely that enumerating screens takes a lot of time, especially now when you use the default config only.

Reply:
It's marked resolved but I can't see any related changes.